introduction.md

```diff
@@ -1,2 +1,8 @@
 Hi, I am @jonico, going through the great Mastering GitHub training
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?